### PR TITLE
Update lib-processing to 3.3.2 and Componenttest to 1.6.3

### PR DESCRIPTION
**Pull request by dattrifork**

### CODE REVIEW

**Review by mnrTrifork — APPROVED** [review]

**Comment by cthtrifork** [comment]
@dattrifork Shouldn't we merge this? :)

**Comment by dattrifork** [comment]
Yes, I've just asked Peter to give it review first.

File: ExternalLookup/ComponentTest/ComponentTest.cs [hunk 1]
```diff
@@ -30,35 +31,33 @@ public ComponentTest()
     }
 
     [Fact]
-    public void Should_BeImplemented_When_ServiceIsCreated()
+    public async Task Should_BeImplemented_When_ServiceIsCreated()
```
Comment (pegtrifork) [inline ExternalLookup/ComponentTest/ComponentTest.cs:34]: Maybe a different name

File: FlinkStates/ComponentTest/ComponentTest.cs [hunk 2]
```diff
@@ -30,12 +31,12 @@ public ComponentTest()
     }
 
     [Fact]
-    public void Should_BeImplemented_When_ServiceIsCreated()
+    public async Task Should_BeImplemented_When_ServiceIsCreated()
```
Comment (pegtrifork) [inline FlinkStates/ComponentTest/ComponentTest.cs:34]: Maybe a different name

File: MergeTwoStreams/ComponentTest/ComponentTest.cs [hunk 3]
```diff
@@ -31,64 +31,63 @@ public ComponentTest()
     }
 
     [Fact]
-    public void Should_BeImplemented_When_ServiceIsCreated()
+    public async Task Should_BeImplemented_When_ServiceIsCreated()
```
Comment (pegtrifork) [inline MergeTwoStreams/ComponentTest/ComponentTest.cs:34]: Maybe a different name

**Review by pegtrifork — APPROVED** [review]
Optional comments

**Pull request merged**